project/src/views/new-evaluation/textarea.js

```diff
@@ -0,0 +1,17 @@
+const Textarea = function(options = {}) {
+    return `<div class="content-box-same">
+            <h2 class="content-box-head content-box-head-skin">${options.title}</h2>
+            <textarea class="content-box-text content-box-text-skin" rows="2">${options.content}</textarea>
+            </div>`;
+  }
+  
+  
+const TextareaSection = function(options = {}) {
+    const section = options.map((t) => Textarea(t))
```

Comment:
I'd argue that this whole method can all be written in a single line of code. Would you believe me? Why / Why not?

Reply:
because map needs a function (luckly Textarea is a function), which has param t, same param as Textarea. so, the answer I knew always (joking, found out one hour ago) is:  options.map(Textarea). Hope you won't ask for a shorter way of writing it further :))